samples/net/sockets/echo_server/boards/atsamr21_xpro.conf

```diff
@@ -1,23 +1,31 @@
 #
-# Copyright (c) 2020-2021, Gerson Fernando Budke
 # Copyright (c) 2019, Benjamin Valentin
+# Copyright (c) 2020-2023, Gerson Fernando Budke
 #
 # SPDX-License-Identifier: Apache-2.0
 #
 
-# Reduced buffers to fit into SAMR21 SoC
+# Reduced configs to fit into SAMR21 SoC
 CONFIG_CPP=n
 
-CONFIG_NET_PKT_RX_COUNT=6
-CONFIG_NET_PKT_TX_COUNT=6
-CONFIG_NET_BUF_RX_COUNT=6
-CONFIG_NET_BUF_TX_COUNT=6
-CONFIG_NET_MAX_CONTEXTS=4
+CONFIG_NET_PKT_RX_COUNT=5
+CONFIG_NET_PKT_TX_COUNT=5
+CONFIG_NET_BUF_RX_COUNT=5
+CONFIG_NET_BUF_TX_COUNT=5
+CONFIG_NET_MAX_CONTEXTS=2
 CONFIG_NET_MAX_CONN=1
 CONFIG_NET_MAX_ROUTES=1
 CONFIG_NET_MAX_NEXTHOPS=1
 
-CONFIG_SHELL_STACK_SIZE=768
-CONFIG_SHELL_CMD_BUFF_SIZE=64
+CONFIG_SHELL_STACK_SIZE=384
```

Comment:
This looks really low but if you have tested that things work ok, then no worries.

Reply:
I'm getting feedback from others that need terminal. I'll return on this soon.